### PR TITLE
Lint fixes in services

### PR DESCRIPTION
**Pull request by mikzat**

### CODE REVIEW

File: frontend/rolecall/src/app/api/cast_api.service.ts [hunk 1]
```diff
@@ -82,278 +84,318 @@ export type OneCastResponse = {
...
+                          return {
+                            uuid: String(rawMem.userId),
+                            position_number: rawMem.order
+                          };
+                        })
+                      });
+                    }
                   } else {
                     groups.push({
-                      position_uuid: String(allPositions.find(pos => Number(pos.uuid) == rawSubCast.positionId).uuid),
```
Comment (yhedholm) [inline frontend/rolecall/src/app/api/cast_api.service.ts:152]: It seems as if the lines at 188 through 201 are the same as 174 through 187.

**Review by yhedholm — CHANGES_REQUESTED** [review]
Thanks for doing this! I believe I caught some code duplication in cast_api.service.ts

File: frontend/rolecall/src/app/api/cast_api.service.ts (hunk 1 above)
Comment (mikzat) [reply frontend/rolecall/src/app/api/cast_api.service.ts:152]: Good catch, I added a TODO above to resolve this later, don't want to mess with logic in the same PR

**Comment by RichardYang2016** [comment]
not familiar yet with logic but just clear for lint fixes.

**Review by RichardYang2016 — APPROVED** [review]

**Review by yhedholm — APPROVED** [review]

**Pull request merged**